getting_started/README.md

```diff
@@ -0,0 +1,73 @@
+# Getting started
+
+Hi and welcome to Crystal's Reference Book!
+
+First we are going to check if we have [installed the language](https://crystal-lang.org/install/) correctly so that we may try all the examples listed in this book.
+
+Once installed, the Crystal compiler should be available as `crystal` command.
+
+Let's try it!
+
+## Crystal version
+
+We may check the Crystal compiler version. If Crystal is installed correctly then we should see something like this:
+
+```terminal-session
+$ crystal --version
+Crystal 0.34.0 (2020-04-07)
```

Comment:
@bcardiff Can we establish a step in the dist process to make sure this is updated on a new release to always show the latest version? I'm not sure if that should just be an item on the manual TODO list or could be an automatic step.

Reply:
Sure, but once we move out from gitbook. To avoid some solution that is might not be the best fit on a future stack.

Reply:
Hm, updating a markdown file should be workable in any case. Or do you have some client-side expansion with current data in mind? That seems like overkill.

Reply:
Currently there is no publish of the book after the release.

For a couple of releases I'm fine manually updating this.

At some point it is another case of how to validate the code samples are still valid.

Having a script that will update the .md in the repo will imply some magic string or ad-hoc process. Between doing that for one case and manually editing it, I prefer the latter.